services/core-web/src/components/common/wrappers/AuthorizationWrapper.tsx

```diff
@@ -1,14 +1,13 @@
 /* eslint-disable */
 import React from "react";
 import ReactDOMServer from "react-dom/server";
-import { PropTypes } from "prop-types";
 import { connect } from "react-redux";
 import { startCase, camelCase } from "lodash";
 import { getUserAccessData } from "@mds/common/redux/selectors/authenticationSelectors";
-import { USER_ROLES } from "@mds/common";
-import { detectDevelopmentEnvironment, detectProdEnvironment } from "@mds/common";
 import { Tooltip } from "antd";
 import * as Permission from "@/constants/permissions";
```

Comment:
I would be inclined to just update to the services/common AuthorizationWrapper instead of using this one.